src/Components/Samples/BlazorUnitedApp/Pages/Index.razor

```diff
@@ -33,7 +33,7 @@
 
     [SupplyParameterFromForm] Customer? Value { get; set; }
 
-    protected override void OnInitialized() => Value ??= new();
+    protected override void OnInitialized() => throw new InvalidOperationException("Some error");
```

Comment:
Nit: revert before merge (I'm sure you were going to anyway)

Reply:
Yeah, this is why I put the PR in draft. Just wanted to showcase the E2E